source/utilities.tex

```diff
@@ -18386,7 +18386,7 @@
 a parallel algorithm's execution may be parallelized.
 \end{itemdescr}
 
-\rSec2[execpol.vec]{Parallel+Vector execution policy}
+\rSec2[execpol.parunseq]{Parallel+Unsequenced execution policy}
```

Comment:
I appreciate that the original wording already does this, but this is typographically gross. Can we do better here, without  a textual `+`?